### ISSUE TITLE
v2.0 Only goes "Whoops" for me...

### ISSUE THREAD

**Issue by iDGS** [edalzell/statamic-profiler-issues#5]
First I tried my code, changing the add-on name as appropriate, but—fail—so I pasted in your sample code as-is from the readme, and got the same thing:
```
Whoops, looks like something went wrong.
1/1 FatalErrorException in ProfilerTags.php line 9:
     Class 'Statamic\Addons\Profiler\Tags' not found 
```

**Comment by iDGS** [comment]
FWIW, permissions are 777 all the way down...

**Comment by iDGS** [comment]
Ed said,
"
add this under the other `use` bits:

`use Statamic\Extend\Tags;`

"
(in ProfilerTags.php) and that fixed this issue.

**Comment by edalzell** [comment]
Fixed in 2.0.1
